src/view/MainWindow.py

```diff
@@ -9,18 +11,44 @@
 from src.view.FileMenu import FileMenu
 from src.view.EditMenu import EditMenu
 
+
 class MainWindow(QMainWindow):
```

Comment:
Class description missing, for example
```suggestion
class MainWindow(QMainWindow):
"""Class containing the widgets for the frontend of the application"""
```